python/src/trezorlib/mintlayer.py

```diff
@@ -18,31 +18,30 @@
 from typing import TYPE_CHECKING, Dict, List, Optional
 
 from . import messages
-from .tools import expect, session
+from .tools import session
 
 if TYPE_CHECKING:
     from .client import TrezorClient
     from .protobuf import MessageType
     from .tools import Address
 
 
-@expect(messages.MintlayerAddress, field="address", ret_type=str)
 def get_address(
     client: "TrezorClient",
     address_n: "Address",
     chain_type: int,
     show_display: bool = False,
     chunkify: bool = False,
-) -> "MessageType":
-
+) -> str:
     return client.call(
         messages.MintlayerGetAddress(
             address_n=address_n,
             chain_type=messages.MintlayerChainType(chain_type),
             show_display=show_display,
             chunkify=chunkify,
-        )
-    )
+        ),
+        expect=messages.MintlayerAddress,
```

Comment:
FYI: this, and the removal of `@expect`, are a fix for the warning `DeprecationWarning: Use client.call(msg, expect=expected) instead`